macros/black_box.init.mac

```diff
@@ -0,0 +1,22 @@
+### --------------------------------------------------------
+### This init macro simulates the response of a SensL SiPM DICE
+### board in the black box.
+### --------------------------------------------------------
+
+/control/execute macros/physics/DefaultPhysicsList.mac
+
+/nexus/RegisterGeometry BlackBox
+#/nexus/RegisterGenerator SingleParticleGenerator
+/nexus/RegisterGenerator LED450LWcone
+/nexus/RegisterPersistencyManager PersistencyManager
+
+/PhysicsList/RegisterPhysics G4OpticalPhysics
+
+### ACTIONS
+
+/nexus/RegisterRunAction DefaultRunAction
+/nexus/RegisterEventAction SaveAllEventAction
```

Comment:
This action doesn't exist anymore, that's why the tests are failing when trying to run this macro.

Reply:
Removed